README.md

```diff
@@ -1,7 +1,62 @@
-# TeamFourHYF
-Recreation of HYF website
+# Project Definition
 
+## About
+
+With related to our course **Incremental Development** module and in practicing of **Planning & Collaboration on software projects**,we here by tried to **Recreate our school's** [website](https://hackyourfuture.be/).
+
+## Technologies
+
+The following tools were used in this project:
+
+- VScode
+- GitHub
+- Git
+- Html
+- CSS
+- Slack
+
+## General Steps
+
+  1. [Create and Setup GitHub Repository](https://github.com/lauraramirez220612/TeamFourHYF)
+  2. [Write Project Definition](README.md)
+  3. [Write Backlog](planning/backlog.md)
+  4. [Make Wire-frame](planning/wireframe.md)
+  5. [Development Strategy](planning/development-strategy.md)
+  6. [Development](https://github.com/lauraramirez220612/TeamFourHYF/blob/main/index.html)
+
+## Screenshots
+
+Intended final result
+
+- [Screenshot image 1](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165146_hackyourfuture.be.jpeg)
+- [Screenshot image 2](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165220_hackyourfuture.be.jpeg)
+- [Screenshot image 3](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165247_hackyourfuture.be.jpeg)
+
+- [Screenshot image 4](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165345_hackyourfuture.be.jpeg)
+
+- [screenshot image 5](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165443_hackyourfuture.be.jpeg)
+
+- [Screenshot image 6](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165512_hackyourfuture.be.jpeg)
+- [Screenshot image 7](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165538_docs.google.com.jpeg)
+
```

Comment:
I clicked on the images links but they opened to a broken link  :thinking:

Reply:
true, although the images are in the repo now..maybe because of the spaces in the file&dir names?

Reply:
thanks for the feedback I will check and fix it